automation/check-patch.e2e.sh

```diff
@@ -29,3 +32,4 @@ main() {
 }
 
 [[ "${BASH_SOURCE[0]}" == "$0" ]] && main "$@"
+
```

Comment:
This is an empty trailing line. It can be dropped.

Reply:
done